React/wbooks/src/redux/books/actions.js

```diff
@@ -0,0 +1,27 @@
+import * as bookService from '../../services/bookService';
+
+import * as types from './actionTypes';
+
+export const fetchBooks = () => async dispatch => {
+  dispatch({ type: types.BOOKS_FETCH });
+  const response = await bookService.getAllBooks();
+  if (response.statusText === 'OK') {
+    dispatch({ type: types.BOOKS_FETCH_SUCCESS, payload: response.data });
+    return response;
+  }
+  dispatch({ type: types.BOOKS_FETCH_FAILURE });
+  return false;
+};
+
+export const saveBookDetailId = bookId => dispatch => {
+  dispatch({ type: types.BOOKS_DETAIL, payload: bookId });
+  dispatch({ type: types.BOOKS_DETAIL_SUCCESS });
```

Comment:
It has no sense manage a `loading` if u justo only update state and aren't fetching data from api. just merge in one action and delete these `isDetailLoading `